src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -199,40 +208,34 @@ internal string GetContent()
             return this.content;
         }
 
-        /// <summary>
-        /// Updates the content to be displayed in the browser and triggers the LinkChanged action.
-        /// </summary>
-        /// <param name="newContent">The content to display.</param>
-        internal void UpdateContent(string newContent)
-        {
-            if (string.IsNullOrWhiteSpace(newContent))
-                return;
-
-            this.content = newContent;
-            this.isRemoteResource = false;
-            OnLinkChanged(null);
-        }
-
-        private void LoadContentFromFile(Uri link)
-        {
-            var path = ResolveFilePath(link);
-            this.content = File.ReadAllText(path);
-
-            // when we have no content fall back to the no content page
-            if (string.IsNullOrWhiteSpace(this.content))
-                throw new TargetException();
-        }
-
         private string LoadContentFromResources(string name)
```

Comment:
can you add a sample of what a valid resource uri looks like?

Reply:
Sure. You can also take a look at the tests for examples.

Basically you have two cases:
- No assembly specified: `href=EmbeddedInDefaultAssembly.html`
- Assembly specified: `href=MyAsssembly;EmbeddedInMyAssembly.html`

The "default assembly" is the documentation browser extension itself.